prowler/__main__.py

```diff
@@ -509,6 +546,18 @@ def prowler():
                     file_path=filename,
                 )
                 cis_finding.batch_write_data_to_file()
+            else:
+                filename = (
+                    f"{global_provider.output_options.output_directory}/compliance/"
+                    f"{global_provider.output_options.output_filename}_{compliance_name}.csv"
+                )
+                generic_compliance = GenericCompliance(
+                    findings=finding_outputs,
+                    compliance=bulk_compliance_frameworks[compliance_name],
+                    create_file_descriptor=True,
+                    file_path=filename,
+                )
+                generic_compliance.batch_write_data_to_file()
```

Comment:
Since this is generic, why not to just add that at the end? Is it necessary to have this logic for each provider?